### ISSUE TITLE
Feature request: add option for different playback speeds

### ISSUE THREAD

**Issue by niklasberlin** [mediathekview/mediathekviewweb#229]
it would be great to have the option to increase or decrease the playback speed of the video like in the official Mediathek. With the used video-js player this should be quite easy as far as I understood the documentation the video-js project offers this functionality out of the box. To enable it, the <video> element just needs an attribute "data-setup" that contains a json object of parameters, the ""playbackRates" parameter that contains a list of possible speeds (for example [0.5, 1, 1.25, 1.5, 2]) should enable the speed selection in the player.

in short: add the following attribute to the video element to enable different playback-speeds:
`data-setup='{ "playbackRates": [0.5, 1, 1.25, 1.5, 2] }'`

relevant topic in the video-js Documentation:
https://docs.videojs.com/tutorial-options.html#playbackrates
